decidim-elections/app/assets/javascripts/decidim/elections/onboarding.js.es6

```diff
@@ -0,0 +1,5 @@
+
+$(async () => {
+  const $onboarding = $("#onboarding-modal");
+  $($onboarding).foundation("open");
```

Comment:
You don't need the `$()` wrapper here, `$onboarding` is already a jQuery object.